Std/Tactic/Omega/Int.lean

```diff
@@ -28,6 +28,8 @@ theorem pos_pow_of_pos (a : Int) (b : Nat) (h : 0 < a) : 0 < a ^ b := by
   rw [Int.eq_natAbs_of_zero_le (Int.le_of_lt h), ← Int.ofNat_zero, ← Int.ofNat_pow, Int.ofNat_lt]
   exact Nat.pos_pow_of_pos _ (Int.natAbs_pos.mpr (Int.ne_of_gt h))
 
+theorem pow_succ (b : Int) (e : Nat) : b ^ e.succ = (b ^ e) * b := rfl
+
```

Comment:
I'm not sure if it matters if this is an omega-specific lemma, but generally I think `e + 1` is preferred over `e.succ`

Reply:
Will be fixed in the patch sent to `leanprover/lean4`.